src/payload/json.cpp

```diff
@@ -564,6 +596,8 @@ int openxc::payload::json::serialize(openxc_VehicleMessage* message,
     size_t finalLength = 0;
     if(root != NULL) {
         bool status = true;
+		if(message->has_uptime)
```

Comment:
A small thing, but the code base always uses curly braces even for 1 line conditionals to avoid errors in the future.